docs/guide/release-notes/1.3.0.md

```diff
@@ -0,0 +1,27 @@
+---
+title: lit-html 1.3.0
+slug: lit-html-1-3-0
+tags: release
+---
+
+See the [changelog](https://github.com/Polymer/lit-html/blob/master/CHANGELOG.md) for a complete list of changes.
+
+## Trusted Types
+
+[Trusted Types](https://web.dev/trusted-types/) helps eliminate cross-site scripting (XSS) attacks by limiting the ability to use unsafe DOM APIs, like `innerHTML`. When trusted types is enabled, unsafe sinks disallow use with easily user-controlled types like strings, and instead require values be one of a small set of trusted types, like `TrustedHTML` that can only be created by a Trusted Type policy.
+
+This prevents application code from accidentally passing unsanitized user-controlled values directly to unsafe sinks and makes it easier to review security sensitive code that uses the Trusted Type policies.
+
+lit-html 1.3.0 is compatible with enabling Trusted Types. lit-html was already very XSS-resistant because it treats strings as _text_ content, not unsafe HTML. It does have to the HTML from template strings to unsafe sinks like `innerHTML`, and now it does this via a Trusted Type policy if the API exists.
+
+Trusted Type shipped in Chrome 83, and is enabled via Content Security Policy. You can (and should!) enable Trusted Types for additional XSS protection on Chrome, and other browsers when they ship Trusted Types.
+
+## "type": "module"
+
+We've added the `"type"` field in package.json, set to "module". This is the standard way to indicate that the .js files in an npm package are standard JavaScript modules. Node only supports this field and not the previous community initiated methods for this. We've kept the existing `"module"` field, since some tools to not yet support the `"type"` field.
+
+## Export `shadyTemplateFactory`
+
+The folk at open-wc.org have an interesting extensions to lit-html that emulates scoped custom element definitions. They had to copy the shady-render though, because `shadyTemplateFactory` wasn't exported. We now export it to save some bytes when using that library.
```

Comment:
```suggestion
The folks at open-wc.org have an interesting extension to lit-html that emulates scoped custom element definitions. They had to copy shady-render though, because `shadyTemplateFactory` wasn't exported. We now export it to save some bytes when using that library.
```

Reply:
"the shady-render" =>  "the shady-render module" or just "shady-render" ...